keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -479,6 +479,11 @@ involve volumes that need to be created by the driver.
 If not set or false, the scheduler makes such decisions without considering
 whether the driver really can create the volumes (the current situation).
 
+This field was initially immutable for the sake of consistency with
+the other fields. Deployments of a CSI driver had to delete and
+recreate the object to switch from "feature disabled" to "feature
+enabled". Starting with Kubernetes 1.23, the field can be modified.
+
```

Comment:
Can you add the motivation for changing this field from "immutable" to "mutable"?  How does this support backward compatibility?  Do we need a feature gate for making this change?

Reply:
> Can you add the motivation for changing this field from "immutable" to "mutable"? How does this support backward compatibility?

I extended the text to address this.

>  Do we need a feature gate for making this change?

No, I don't think so and Jordan seemed to agree when reviewing the API change.

Reply:
Is there a PR for this change?

Reply:
Ok.  I found it. https://github.com/kubernetes/kubernetes/pull/101789

Reply:
Yes: https://github.com/kubernetes/kubernetes/pull/101789

Reply:
And also an issue: https://github.com/kubernetes/kubernetes/issues/100089